src/main/webapp/WEB-INF/spring-servlet.xml

```diff
@@ -27,7 +27,7 @@
 
 
     <bean id="ds" class="org.springframework.jdbc.datasource.DriverManagerDataSource">
-        <property name="driverClassName" value="com.mysql.jdbc.Driver"/>
+        <property name="driverClassName" value="com.mysql.cj.jdbc.Driver"/>
```

Comment:
What kind of driver is it?

Reply:
It's the newer version of the same driver